cmd/create/cluster/cmd.go

```diff
@@ -655,6 +655,11 @@ func getVersionList(client *cmv1.Client, channelGroup string) (versionList []str
 		versionList = append(versionList, strings.Replace(v.ID(), "openshift-v", "", 1))
 	}
 
+	if len(versionList) == 0 {
+		err = fmt.Errorf("Failed to find versions for the provided channel-group: '%s'.", channelGroup)
```

Comment:
This has a linting error:
```
cmd/create/cluster/cmd.go:659:20: error strings should not be capitalized or end with punctuation or a newline (golint)
		err = fmt.Errorf("Failed to find versions for the provided channel-group: '%s'.", channelGroup)
		                 ^
make: *** [Makefile:45: lint] Error 1
```

Reply:
Yeah I see - but the error just above it is capitalized :sad